multidb/__init__.py

```diff
@@ -38,29 +39,66 @@
 DEFAULT_DB_ALIAS = 'default'
 
 
-if getattr(settings, 'SLAVE_DATABASES'):
-    # Shuffle the list so the first slave db isn't slammed during startup.
-    dbs = list(settings.SLAVE_DATABASES)
+def _get_replica_list():
+    dbs = None
+    if getattr(settings, 'REPLICA_DATABASES'):
+        dbs = list(settings.REPLICA_DATABASES)
+    elif getattr(settings, 'SLAVE_DATABASES'):
+        warnings.warn(
+            '[multidb] The SLAVE_DATABASES setting has been deprecated. '
+            'Please switch to the REPLICA_DATABASES setting.',
+            DeprecationWarning,
+        )
+        dbs = list(settings.SLAVE_DATABASES)
+
+    if not dbs:
+        return itertools.repeat(DEFAULT_DB_ALIAS)
+
+    # Shuffle the list so the first replica isn't slammed during startup.
     random.shuffle(dbs)
-    slaves = itertools.cycle(dbs)
-    # Set the slaves as test mirrors of the master.
+
+    # Set the replicas as test mirrors of the master.
     for db in dbs:
         settings.DATABASES[db].get('TEST', {})['MIRROR'] = DEFAULT_DB_ALIAS
-else:
-    slaves = itertools.repeat(DEFAULT_DB_ALIAS)
+
+    return itertools.cycle(dbs)
+
+
+replicas = None
+if replicas is None:
+    replicas = _get_replica_list()
```

Comment:
Also considering doing this caching inside `_get_replica_list()` and just calling it more often. It'd even be possible to add a private cache-clearer for use in tests e.g. with `@override_settings`

Reply:
That sounds like a good idea as well, but maybe overkill for this PR.

Reply:
I think I'll move the caching but not necessarily update tests right now